skins/oasis/modules/templates/LatestActivity_Index.php

```diff
@@ -1,27 +1,20 @@
-<section class="WikiaActivityModule module" id="<?= !empty( $userName ) ? 'WikiaRecentActivityUser' : 'WikiaRecentActivity'; ?>">
-	<h2 class="activity-heading"><?= $moduleHeader ?></h2>
-	<ul>
-<?php
-	if(!empty($changeList)){
-		foreach ($changeList as $item) {
-?>
-		<li>
-			<img src="<?= $wg->BlankImgUrl ?>" class="sprite <?= $item['changeicon'] ?>" height="20" width="20">
-			<em><?= $item['page_href'] ?></em>
-			<div class="edited-by"><?= $item['changemessage'] ?></div>
-		</li>
-<?php
-		}
-	}
-	elseif(!empty($userName)) {
-		echo wfMsg( 'userprofilepage-recent-activity-default', $userName );
-	}
-?>
-	</ul>
+<section class="module" id="WikiaRecentActivity">
 
-	<? if ( $userName && count($changeList) ) :?>
-		<?= Wikia::specialPageLink('Contributions/' . $userName, 'userprofilepage-top-recent-activity-see-more', 'more') ;?>
-	<? elseif(empty($userName)): ?>
-		<?= Wikia::specialPageLink('WikiActivity', 'oasis-more', 'more') ?>
-	<? endif ;?>
+	<h2 class="activity-module-header"><?= $activityIcon ?><?= $moduleHeader ?></h2>
+
+	<? if ( !empty( $changeList ) ): ?>
+		<ul class="activity-items">
+			<? foreach ( $changeList as $item ): ?>
+				<li class="activity-item">
+					<div class="page-title"><a href="<?= $item['page_url'] ?>" data-tracking="activity-title" ><?= $item['page_title'] ?></a></div>
+					<div class="edit-info">
+						<a class="edit-info-user" data-tracking="activity-username" href="<?= $item['user_profile_url'] ?>"><?= $item['user_name'] ?></a>
```

Comment:
It's safer when we escape data that can be provided by user like `user_name` and others.